### DISCUSSION TITLE
Is anyone successfully building with Visual Studio...?

### DISCUSSION THREAD

**Discussion by jamiebullock** [MTG/essentia#1361] [General]
There is a big thread [here](https://github.com/MTG/essentia/issues/330) with various people trying to get Essentia to build on Windows.

There is also mention of a [python_win](https://github.com/MTG/essentia/tree/python_win) branch, which appears to have been updated a few years ago.

Anyone know of the latest status of all this?

I've managed to get an `essentia.lib` to build with a whole bunch of minor tweaks and a hack to `RogueVector`, but wanted to check if anyone else has it building before I go any further so as not to duplicate effort.

Anyone...?

**Comment by regorxxx** [comment]
I don't think this can be build for windows at all.
https://github.com/MTG/essentia/issues/1340

I don't really see the point of a repository being constantly updated which can not be built by mortals with zero support, outdated docs and thousands of threads with the same issue..

At this point if someone can provide binaries for windows + Gaia It would be a pleasure to pay to that person.
